hunter/config.py

```diff
@@ -85,13 +85,22 @@ def load_config_from(config_file: Path) -> Config:
         if Grafana configs not explicitly set in yaml file, default to same as Graphite 
         server at port 3000
         """
-        if config["graphite"]["url"] is None:
-            raise ValueError("graphite.url")
-        if config.get("grafana") is None:
-            config["gafana"] = {}
-            config["grafana"]["url"] = f"{config['graphite']['url'].strip('/')}:3000/"
-            config["grafana"]["user"] = os.environ.get("GRAFANA_USER", "admin")
-            config["grafana"]["password"] = os.environ.get("GRAFANA_PASSWORD", "admin")
+        graphite_config = None
+        grafana_config = None
+        if "graphite" in config:
+            if "url" not in config["graphite"]:
+                raise ValueError("graphite.url")
+            graphite_config = GraphiteConfig(url=config["graphite"]["url"])
+            if config.get("grafana") is None:
+                config["gafana"] = {}
```

Comment:
Fixed in the last commit. It is broken, but I think we haven't hit it because all of the configs I've seen / used explicitly define a grafana section.